music21/note.py

```diff
@@ -1445,7 +1462,7 @@ def _setOctave(self, value: int):
     @property
     def pitches(self) -> Tuple[pitch.Pitch]:
         '''
-        Return the :class:`~music21.pitch.Pitch` object in a tuple.
+        Return the :class:`~music21.pitch.Pitch` objects in a tuple.
```

Comment:
maybe clearer "Return the single Pitch object within a tuple" -- there can't be more than one pitch object here.